packages/core/admin/admin/src/components/CommandPalette/Items.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useCommand } from './context';
+import Item from './Item';
+
+const Items = ({ items, displayOnSearchOnly }) => {
+  const cmd = useCommand();
+
+  return (
+    <>
+      {items.map((item, idx) => {
+        if (item.component) {
+          return React.cloneElement(item.component, { key: idx });
+        }
+
+        return (
+          <Item
+            key={`${item.label}-${idx}`}
```

Comment:
The label isn't unique that's why I added the index even if as you say it's not stable

Reply:
How come the label isn't unique? 🤔 I don't see any duplicates in what we have now.

Reply:
Ha you are right, I think I added some extra words but I just had the ct.info.dsiplayname at 1st. considering this could still happen if we add some things would you still use the label alone ? I'm fine with that

Reply:
yeah I would use the label alone for now, if we run into issues we can re-evaluate! 😄